packages/python/vc_init.py

```diff
@@ -191,23 +192,33 @@ def __init__(self, scope):
                 self.state = ASGICycleState.REQUEST
                 self.app_queue = None
                 self.response = {}
+                self._legacy_asyncio = "loop" in inspect.signature(asyncio.Queue).parameters
```

Comment:
I am not a fan of using `inspect` here. We know for a fact that, for example, `loop` was removed in Python 3.10. So, instead of this field, I'd compare `sys.version_info >= (3, 10)`

Reply:
thanks, I've updated the check.